internal/cmd/fetcher/main.go

```diff
@@ -144,14 +146,19 @@ func runPluginTests(plugins []createdPlugin, rootDir string) error {
 		Args: []string{
 			makePath,
 			"test",
-			fmt.Sprintf("PLUGINS=%s", strings.Join(pluginsEnv, ",")),
+			fmt.Sprintf("PLUGINS=%s", pluginRef),
 		},
-		Dir:    rootDir,
-		Stdout: os.Stdout,
-		Stderr: os.Stderr,
-		Env:    env,
+		Dir: baseDir,
+		Env: env,
 	}
-	return cmd.Run()
```

Comment:
Maybe I'm misunderstanding how this works, but from trying this locally it masks the error, e.g.,

```bash
make: *** No rule to make target `test'.  Stop.
failed to run post-processing on plugins: exit status 2exit status 1
```

Whereas when you do `CombinedOutput()` and grab the returned string (even on an error) you get the underlying output, e.g.,

```
failed to run post-processing on plugins: failed to run plugin tests: exit status 2
docker buildx inspect "bufbuild-plugins" 2> /dev/null || docker buildx create --use --bootstrap --name="bufbuild-plugins"
ERROR: existing instance for "bufbuild-plugins" but no append mode, specify --node to make changes for existing instances
make: *** [build] Error 1
exit status 1
```